ajax/libs/jQuery.Gantt/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "jQuery.Gantt",
+  "version": "1.2.2",
+  "description": "jQuery Gantt Chart is a simple chart that implements gantt functionality as a jQuery component.",
+  "filename": "js/jquery.fn.gantt.min.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/mbielanczuk/jQuery.Gantt.git"
+  },
+  "keywords": [
+    "jquery",
+    "gantt",
+    "chart"
+  ],
+  "author": {
+    "name": "Marek Bielańczuk",
+    "url": "https://github.com/mbielanczuk"
+  },
+  "licenses": [
+    "MIT",
+    "GPL-2.0"
+  ],
+  "homepage": "https://mbielanczuk.com/jquery-gantt/",
```

Comment:
Yes, it can show the page.
But what I consider is using `https` is a way to prevent the possibility the users been attacked during the redirection.
And in this case, the page has SSL certificate problem, then I think it not a secure way.

@PeterDaveHello could you give us some suggestions?
Thank you.